error.js

```diff
@@ -0,0 +1,17 @@
+exports.forCustomErrors = (err, req, res, next) => {
+    console.log("hce");
+    if (err.status && err.msg) {
+      res.status(err.status).send({ msg: err.msg });
+    } else {
+      next(err);
+    }
+  }
+
+  exports.forPgErrors = (err, req, res, next) => {
+    if (err.code === "22P02") {
+      console.log("I worked");
+      res.status(400).send({ msg: "Invalid Input" });
```

Comment:
I think this error message could be something different since it's not really dealing with an input, bad request would be fine